### PR TITLE
 Stickers: Clean up panel

### LINKED ISSUE #6187: Stickers: Clean up panel
## Feature Description

Currently, our shapes panel only includes basic shapes which also act as masks. We'd like to expand this section to include stickers - svg elements that are not maskable. Many of these will be used in the new templates we are building. 

The request in this ticket is to:

1. Update the shapes panel so that it clearly differentiates between shapes and stickers
2. Load up the initial set of stickers into it (the ones needed to build our upcoming templates)

~@samitron7 adding this to the UX pipeline for the designs on how shapes and stickers would be seen. We should also attach the list of SVGs/Stickers that will be needed for the templates (it's OK if it's a superset and we end up using less).~

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief (Updated 7/17/2021)

- Follow the design system panel of 'shapes' to implement the design for stickers. 
- Stickers will be kept separate from shapes but share a panel (as they currently do) 
- Make sure that stickers are navigable via keyboard and no focus trapping occurs. 
- Stickers should be draggable 
- Stickers should be able to click to be added 
- Add tests accordingly 

While right now we are not implementing a 'search' or a filter, do keep in mind that that will be happening later based on sticker metadata - a bit like the text sets. 

Further implementation of this feature will be scoped as its own tickets.

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Part of exposing stickers panel to users

## Summary
Hits these implementation brief points for the stickers panel:
- Follow the design system panel of 'shapes' to implement the design for stickers.
- Stickers will be kept separate from shapes but share a panel (as they currently do)
- Make sure that stickers are navigable via keyboard and no focus trapping occurs.
- Stickers should be draggable
- Stickers should be able to click to be added
- Add tests accordingly

## Relevant Technical Choices
Predominately just followed existing practices and setup in the existing shapes part of the panel

## To-do
- maybe cleanup shareable constants a little 🤷 

## User-facing changes
NA

## Testing Instructions
- Go to experiments and enable `Stickers`
- Open a story in the editor
- go to the shapes panel and you should now see a library of stickers under the shapes
- visually check them, keyboard navigate through them and see that they drag n drop just like the shapes in the shapes panel

### QA
- Go to experiments and enable `Stickers`
- Open a story in the editor
- go to the shapes panel and you should now see a library of stickers under the shapes
- visually check them, keyboard navigate through them and see that they drag n drop just like the shapes in the shapes panel

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6187

### CODE REVIEW

**Review by BrittanyIRL — COMMENTED** [review]
This is awesome! 

I think a few little things might be helpful here, but over all two thumbs up 👍 👍  

1. Thoughts on updating the "Shapes" tooltip to be "Shapes and Stickers" (I know this wasn't in the brief, we're kinda ad-libbing a bit, i'm sorry) 
Screen Shot 2021-07-27 at 4 24 24 PM

2. I think adding karma tests for stickers would be a solid move, that way we can test the accessibility there as well which would be nice.

**Review by samwhale — APPROVED** [review]
noice, the stickers panel works great!

File: assets/src/edit-story/components/library/constants.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const TEXT = {
 };
 export const SHAPES = {
   icon: ShapesIcon,
-  tooltip: __('Shapes', 'web-stories'),
+  tooltip: __('Shapes & Stickers', 'web-stories'),
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/constants.js:53]: 💯  thank you!

File: assets/src/edit-story/components/library/karma/shapes/shapes.karma.js [hunk 2]
```diff
@@ -86,3 +93,97 @@ describe('Shape library integration', () => {
...
+      moveRel(stickerButton, 10, 10),
+      down(),
+      /* The steps give time for Moveable to react and display a clone to drag */
+      moveRel(bgFrame, -(pageWidth / 3 + 20), 50, { steps: 20 }),
+      up(),
+    ]);
+
+    // Still only background.
+    expect(fixture.editor.canvas.framesLayer.frames.length).toBe(1);
+  });
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/library/karma/shapes/shapes.karma.js:188]: nit: add one more check? Something like this. It's a habit we're trying to get into on new features.  
```
it('should not have any accessibility violations', async () => {
 await expectAsync(fixture.editor.library.shapes.getByTestId('stickers-library-pane')).toHaveNoViolations();
})
```

**Review by BrittanyIRL — APPROVED** [review]
Thanks for the updates! Looks good to me, requested one more test to add but after that this seems super done.

**Pull request merged**